public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Question.js

```diff
@@ -19,29 +23,93 @@ export class StoryQuestionsQuestion extends React.Component {
     onFormErrorsUpdate: PropTypes.func
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      answers: props.fieldValue && props.fieldValue.answers && props.fieldValue.answers.length ?
```

Comment:
As lodash is included, you can make this a lot more terse using `_.get()`

Reply:
Would this cause us to lose any checks done by the linter?
(I don't know if the linter can even check this; excuse my ignorance!)
```

Reply:
An alternative (for the fun of code golf):
```javascript
answers: (props.fieldValue || {}).answers || []
```

Reply:
@hosamaly lodash would be `_.get(props.fieldValue, 'answers')` I think. Not sure about the linter, depends on rules but it shouldn't, no